db/migrate/20150629192719_add_password_reset_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddPasswordResetToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :password_reset_token, :string
+    add_column :users, :password_reset_sent_at, :datetime
+    add_index :users, :password_reset_token
```

Comment:
Looks like you added the index, but it's not showing up in the `db/schema.rb` changes. Make sure you rollback this migration and run it again so that gets picked up.

(You'll want to comment out line 5 before you rollback and uncomment after, otherwise it will try to rollback the `add_index` and it will fail because the index doesn't exist.)